x/distribution/keeper/genesis.go

```diff
@@ -123,9 +123,6 @@ func (k Keeper) InitGenesis(ctx context.Context, data types.GenesisState) error
 	}
 
 	balances := k.bankKeeper.GetAllBalances(ctx, moduleAcc.GetAddress())
-	if balances.IsZero() {
```

Comment:
What was the reason for this and what does it mean to remove it?

Reply:
A few lines above `GetDistributionAccount` is called, which calls `GetModuleAccount`, then `GetModuleAccountAndPermissions` which calls `SetModuleAccount`. So this is not needed